FirmataParser.cpp

```diff
@@ -163,8 +166,8 @@ void FirmataParser::parse(uint8_t inputData)
         systemReset();
         break;
       case REPORT_VERSION:
-        if (currentReportVersionCallback)
-          (*currentReportVersionCallback)(currentReportVersionCallbackContext);
+        waitForData = 2; // two data bytes needed
```

Comment:
This is because many Firmata client libraries initiate communication with the board by querying the protocol or version.